quarkus/runtime/src/main/java/org/keycloak/configuration/PropertyMappers.java

```diff
@@ -25,6 +25,7 @@
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
+import com.google.common.base.Ascii;
```

Comment:
Nitpick: There is no need for this import? It seems it is not used anywhere. But I don't consider this as a PR blocker :)